ext/http/http_next.rs

```diff
@@ -1088,11 +1100,15 @@ impl Resource for UpgradeStream {
   }
 }
 
-#[op(fast)]
-pub fn op_can_write_vectored(state: &mut OpState, rid: ResourceId) -> bool {
+#[op2(fast)]
+pub fn op_can_write_vectored(
+  state: &mut OpState,
+  #[smi] rid: ResourceId,
+) -> bool {
   state.resource_table.get::<UpgradeStream>(rid).is_ok()
 }
 
+// TODO(bartlomieju): op2 doesn't want to handle `usize` in the return type
```

Comment:
In this case it can be #[bigint] usize or #[bigint] u64, I think.

Reply:
I believe in this case we should cast it to `u32` before returning. The JS side expects a `number`, not `BigInt`

Reply:
Filed https://github.com/denoland/deno_core/issues/107 to make this `Number`-safe.